tests/fixture_island_nesting/options.ts

```diff
@@ -0,0 +1,11 @@
+import { FreshOptions } from "$fresh/server.ts";
+
+export default {
+  async render(_ctx, render) {
```

Comment:
Is this `render` function useful for anything, or just copy paste?

Reply:
err... just copy & paste 😅